### ISSUE TITLE
Fail installation if execute:command fails

### ISSUE THREAD

**Issue by rockerbacon** [lutris/lutris#2852]
**Describe the bug**

When executing a custom command with execute:command, Lutris will continue the installation even if the command exits with a non-zero code.

**Expected behavior**

If a command is issued it's because its execution is essential for ensuring a correct installation. It's easier for the user to be informed of problems as soon as they occur than to hunt down the cause of a problem later on.

**Current behavior**

Lutris continues installation even after commands fail.

**Steps to reproduce**

The following installer can be used to reproduce the issue:

```yml
name: Reproduce issue
game_slug: reproduce-issue
slug: reproduce-issue
version: 1.0.0 issue
runner: linux

script:
    game:
        exe: /bin/bash

    installer:
    - execute:
        command: "echo ERROR; exit 1"

    - input_menu:
        description: Installation continues even when error occurs
        options:
          - continue: continue
```

**Comment by strycore** [comment]
I wasn't able to find a solution for that. The return code for the monitored command is always 0, even when the subprocess has exited with something else.

**Comment by strycore** [comment]
This needs proper UI but it is otherwise fixed.

**Comment by strycore** [comment]
Closed in 0.5.9
